internal/driver/driver.go

```diff
@@ -150,11 +150,11 @@ func generateReport(p *profile.Profile, cmd []string, vars variables, o *plugin.
 }
 
 func applyCommandOverrides(cmd []string, v variables) variables {
-	trim, focus, tagfocus, hide := v["trim"].boolValue(), true, true, true
+	trim, tagfocus, filter := v["trim"].boolValue(), true, true
```

Comment:
Maybe rename "tagfocus" to "tagfilter", I think it will be clearer, too.

Reply:
done.